easybuild/easyconfigs/l/LASTZ/LASTZ-1.02.00-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'LASTZ'
+version = '1.02.00'
+
+homepage = 'http://www.bx.psu.edu/~rsharris/lastz/'
+description = """ LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to handle
+ sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS 
+ sequencing technologies such as Roche 454.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.bx.psu.edu/miller_lab/dist/']
+
+patches = ['LASTZ-%(version)s_Makefile.patch']
```

Comment:
please add the missing SHA256 checksums (`eb --inject-checksums` is your friend here)